app.json

```diff
@@ -15,7 +15,13 @@
   },
   "env": {
     "JOSS_API_KEY": {
-      "description": "The API key for the JOSS API"
+      "description": "The API key for the JOSS site API"
+    },
+    "JOSE_API_KEY": {
```

Comment:
Don't love the fact that we've got these configs in the `app.json` but it does mean we don't need any hidden config files (e.g. `.env-production`